README.md

```diff
@@ -1002,6 +1002,7 @@ See https://staticsitegenerators.net and https://www.staticgen.com
   * [Restyaboard](http://restya.com/board/) - Open source Trello-like kanban board. ([Demo](http://restya.com/board/demo.html), [Source Code](https://github.com/RestyaPlatform/board)) `OSLv3` `PHP`
   * [TaskBoard](http://taskboard.matthewross.me/) - A Kanban-inspired app for keeping track of things that need to get done. ([Source Code](https://github.com/kiswa/TaskBoard)) `MIT` `PHP`
   * [Taskfreak](http://www.taskfreak.com/original) - A simple but efficient web based task manager written in PHP. ([Demo](http://demo.taskfreak.com/)) `GPLv3` `PHP`
+  * [Tasks](https://github.com/m1guelpiedrafita/Tasks) - Simple tasks & notes manager written in PHP, jQuery and Bootstrap using a custom flat file database. ([Demo](http://projects.miguelpiedrafita.com/Tasks/) `MPLv2` `PHP`
```

Comment:
Please change the homepage URL to `https://projects.miguelpiedrafita.com/Tasks/`, change the Demo URL to `https://demo.miguelpiedrafita.com/tasks/` and add the GitHub URL in a Source Code section.